Signal/src/environment/NotificationsManager.m

```diff
@@ -263,8 +263,13 @@ - (void)notifyUserForIncomingMessage:(TSIncomingMessage *)message
         }
 
         switch (self.notificationPreviewType) {
-            case NotificationNamePreview:
-                notification.category = Signal_Full_New_Message_Category;
+            case NotificationNamePreview: {
+                BOOL senderIsNoLongerVerified =
+                    [OWSIdentityManager.sharedManager verificationStateForRecipientId:message.authorId]
+                    == OWSVerificationStateNoLongerVerified;
```

Comment:
What if it's an incoming group message, and someone *else* is no longer verified?